app/constants/sharedPreferences.js

```diff
@@ -0,0 +1,6 @@
+export const SHARED_PREFERENCES_ACCOUNTS_KEY = 'ACCOUNTS_KEY';
```

Comment:
What do you think about use ternary condition in this file and mantain same variables?
Example: `const SERVER = isIOS ? 'kAuthServer' : 'authServer'`
https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/constants/userDefaults.js
Probably it help us to make only one function to migrate data both iOS and Android.

Reply:
We can change filename.

Reply:
@djorkaeffalexandre We are now calling the filename as `native.js` instead and keeping everything related to native credentials in this single file - it is better (thx for the suggestion). Let me know if you have a better file name suggestion too :)

Reply:
@djorkaeffalexandre @filipedelimabrito Of course we are using same keys as native and changed our to mirror those, but feels weird to have `native.js` in RN.
In a year from now, we'll have this file and have to think "oh, this is a trace from that migration".

Don't you guys think `credentials.js` or something like that is better?

Reply:
Changed!